docs/how-to-guides/block-tutorial/extending-the-query-loop-block.md

```diff
@@ -0,0 +1,195 @@
+
+In the same manner, a lot of extenders might need a way to present bespoke versions of the block, with their own presets, additional settings and without customization options which are irrelevant to their use-case (often, for example, their custom post type). The Query Loop block offers very powerful ways to create such variations.
+
+## Extending the block with variations
+
+By registering your own block variation with some specific Query Loop block settings, you can have finer control over how it is presented, while still being able to use the full capabilities which the Query Loop block offers underneath. If you are not familiar with block variations, learn more about them [here](/docs/reference-guides/block-api/block-variations.md).
+
+With the block variations API you can provide the default settings that make the most sense for your use-case.
+
+Let's go on a journey, for example, of setting up a variation that could fit a plugin which registers a `book` [custom post type](https://developer.wordpress.org/plugins/post-types/).
+
+### Offer sensible defaults
+
+Your first step would be to create a variation which will be set up in such a way to provide a block variation which will display by default a list of books instead of blog posts. You would start with something like this:
+
+```js
+registerBlockVariation( 'core/query', {
+	name: 'my-plugin/books-list',
+	attributes: {
+		query: {
+			/** ...more query settings if needed */
+			postType: 'book',
+		},
+	},
+} );
+```
+
+In this way, the users won't have to choose the custom `postType` from the dropdown, and be already presented with the correct configuration. However, you might ask, how is a user going to find and insert this variation? Good question! To enable this, you should add:
+
+```js
+{
+	/** ...variation properties */
+	scope: [ 'inserter' ],
+}
+```
+
+In this way, your block will show up just like any other block while the user is in the editor and searching for it. At this point you might also want to add a custom icon, title and description to your variation, just like so:
+
+```js
+{
+	/** ...variation properties */
+	title: 'Books List',
+	description: 'Displays a list of books',
+	icon: /* Your svg icon here */,
+}
+```
+
+Also note that the Query Loop block supports `'block'` as a string in the `scope` property, and it will allow the variation to be picked up after inserting the block itself. Read more about the Block Variation Picker [here](https://github.com/WordPress/gutenberg/blob/HEAD/packages/block-editor/src/components/block-variation-picker/README.md).
```

Comment:
Yeah, I think we should skip it. With https://github.com/WordPress/gutenberg/pull/44197 a plugin should either register their own patterns for best user experience or just add `innerBlocks` to their variations and have the same result in every insertion. Of course it would preferred I think to register patterns..

The `scope` variations are really simple variations that few would select, if they have something specific in mind they want to build. I also chatted with Luigi(Woo Blocks) a bit earlier and it seems you had that scope because it was not possible to do it better - before the above mentioned PR. I don't think you should use that scope to be honest and I think the best `scope` for such variations would be the `inserter` one.